CASSANDRA-18185 Accumulate all `docs` PR (#2062)
apache/cassandra: doc/modules/cassandra/pages/architecture/dynamo.adoc

Transient replication is an experimental feature that is not ready
for production use. The expected audience is experienced users of
Cassandra capable of fully validating a deployment of their particular

> application. That means being able to check that operations like reads,

Reviewer: ```suggestion
application. That means you have the experience to check that operations like reads,
```
